examples/config/global_config.yaml

```diff
@@ -1,21 +1,136 @@
 blob-perf-buffer-size: 1024
 cache:
-    - none
+    type: none
+    # size: 1024
+
 proctree:
-    - none
-capabilities: []
+    source: none
+    # cache:
+    #     process: 8192
+    #     thread: 4096
+
+capabilities:
+    bypass: false
+    # add:
+    #     - cap_sys_admin
+    #     - cap_syslog
+    # drop:
+    #     - cap_chown
+
 containers: false
-crs: []
+crs:
```

Comment:
We should rename this to cri, which is much more clear (can be in a different PR)

Reply:
Done: 33686e61ac786002789b4ae560c8bd685f14e972